gdshelpers/helpers/small.py

```diff
@@ -47,6 +48,20 @@ def id_to_alphanumeric(column, row):
     return int_to_alphabet(row) + str(int(column))
 
 
+def parse_alphanumeric(text):
```

Comment:
to keep it consistent, I'd suggest a similar name to `id_to_alphanumeric', I'd also move it to the same file as `id_to_alphanumeric' (and directly below that function)
```suggestion
def alphanumeric_to_id(text):
```